docs/reference/announcements-release-notes/870/870-announcements.md

```diff
@@ -1,53 +1,45 @@
 
-| Scheduled release date | Scheduled end of maintenance | Release notes                                        | Blog                                                                                            |
-| :--------------------- | :--------------------------- | :--------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
-| 8 April 2025           | 13 October 2026              | [8.7 release notes](/reference/release-notes/870.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
+| Scheduled release date | Scheduled end of maintenance | Release notes                                                                        | Blog                                                                                            |
+| :--------------------- | :--------------------------- | :----------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
+| 8 April 2025           | 13 October 2026              | [8.7 release notes](/reference/announcements-release-notes/870/870-release-notes.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
 
 <!--- [Ad-hoc subprocesses](#)
 - [Document handling](#)
 - [RPA](#)
   - [Fetch RPA resource API](#)
   - [deployResourceAPI for RPA](#) -->
 
-- [Amazon Web Services (AWS) region support](#amazon-web-services-aws-region-support-saas)
-- [Web Modeler](#web-modeler)
-- [Southeast Asia region for SaaS customers](#southeast-asia-region-for-saas-customers-saas)
-- [Breaking changes in Camunda Process Test](#breaking-changes-in-camunda-process-test)
-- [API updates](#api-updates-saasself-managed)
-- [Identity Keycloak now requires v25 or v26](#identity-keycloak-now-requires-v25-or-v26-self-managed)
-- [Camunda Spring SDK now requires Spring Boot 3.4.x](#camunda-spring-sdk-now-requires-spring-boot-34x-saasself-managed)
-- [Zeebe Java client](#zeebe-java-client)
+- [Changes in supported environments](#changes-in-supported-environments)
+  - [Identity Keycloak now requires v25 or v26 Self-Managed](#identity-keycloak-now-requires-v25-or-v26-self-managed)
+  - [Camunda Spring SDK now requires Spring Boot 3.4.x SaaSSelf-Managed](#camunda-spring-sdk-now-requires-spring-boot-34x-saasself-managed)
+- [Key changes](#key-changes)
+  - [Deploy diagram change Self-Managed](#web-modeler-deploy-diagram-change)
+  - [Breaking changes in Camunda Process Test](#breaking-changes-in-camunda-process-test)
+  - [Deprecated: OpenAPI entities with `integer (int64)` key attributes](#deprecated-openapi-entities-with-integer-int64-key-attributes)
+  - [Zeebe Java client](#zeebe-java-client)
+    - [Adjustments](#adjustments)
```

Comment:
Just flagging that the ToC still exists in the 8.7 announcements - the other pages all look good!